defaults/main.yml

```diff
@@ -62,14 +62,14 @@ vault_main_configuration_template: vault_main_configuration.hcl.j2
 # Storage backend
 # ---------------------------------------------------------------------------
 
-vault_backend: consul
 vault_backend_consul: vault_backend_consul.j2
 vault_backend_file: vault_backend_file.j2
 vault_backend_raft: vault_backend_raft.j2
 vault_backend_etcd: vault_backend_etcd.j2
 vault_backend_s3: vault_backend_s3.j2
 vault_backend_dynamodb: vault_backend_dynamodb.j2
 vault_backend_mysql: vault_backend_mysql.j2
+vault_backend_raft: vault_backend_raft.j2
```

Comment:
**issue**: you seem to have a duplicate of line 67

Reply:
SHoud be good now.